@uportal/esco-content-menu/src/components/ContentGridFilter.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="grid-categories">
+    <select
+      @change="fireFilterCategoryEvent()"
+      class="footer-categories"
+      v-model="filterCategory">
+      <option
+        class="default"
+        selected
+        value="">
+        {{ translate('message.filter.selectOption') }}
+      </option>
+      <option
+        v-for="category in allCategories"
+        :value="category"
+        :label="category"
+        :key="category">
+        {{ category }}
+      </option>
+    </select>
+  </div>
+</template>
+
+<script>
+import i18n from '../i18n.js';
+
+export default {
+  name: 'ContentGridFilter',
+  data() {
+    return {
+      filterCategory: '',
+      allCategories: [],
+    };
+  },
+  beforeMount() {
+    window.addEventListener('gridCategories', this.setAllCategories);
+  },
+  mounted() {
+    this.fireCategoriesRequest();
+  },
+  beforeDestroy() {
+    window.removeEventListener('gridCategories', this.setAllCategories);
+  },
+  methods: {
+    translate(text, lang) {
+      return i18n.t(text, lang);
+    },
+    fireCategoriesRequest() {
+      this.$nextTick(() => {
+        // await for data to resolve
+        if (this.allCategories && this.allCategories.length > 0) {
```

Comment:
```suggestion
        if (this?.allCategories?.length > 0) {
```

Reply:
No issues with IE11 compatibility?

Reply:
No issues with IE11 compatibility?

I wouldn't think so, b/c it'll be transpiled by Babel (or similar).

Reply:
Babel rewrites it back to ES2009, IE11 compatibility is good :ballot_box_with_check:

Reply:
I couldn't tell if it was still a proposal or production-ready.